src/Examples/UnitModels/Operations/skeleton_unit_testing.ipynb

```diff
@@ -0,0 +1,750 @@
+    "\n",
+    "# internal vars for skeleton\n",
+    "energy_activation_dict = {(0, 'Liq', 'water'): 51e3, (0, 'Liq', 'ethylene_glycol'): 53e3}\n",
+    "m.fs.pervap.energy_activation = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
+    "                                    initialize=energy_activation_dict, units=pyunits.J/pyunits.mol)\n",
+    "m.fs.pervap.energy_activation.fix()\n",
+    "\n",
+    "permeance_dict = {(0, 'Liq', 'water'): 5611320, (0, 'Liq', 'ethylene_glycol'): 22358.88}  # calculated from literature data\n",
+    "m.fs.pervap.permeance = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
+    "                            initialize=permeance_dict, units=pyunits.mol/pyunits.s/pyunits.m**2)\n",
+    "m.fs.pervap.permeance.fix()\n",
+    "\n",
+    "m.fs.pervap.area = Var(m.fs.time, initialize=6, units = pyunits.m**2)\n",
+    "m.fs.pervap.area.fix()\n",
+    "\n",
+    "cond_heat_dict = {(0, 'Liq', 'water'): 40.660e3, (0, 'Liq', 'ethylene_glycol'): 56.9e3}\n",
+    "m.fs.pervap.cond_heat = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
+    "                            initialize=cond_heat_dict, units=pyunits.J/pyunits.mol)\n",
+    "m.fs.pervap.cond_heat.fix()\n",
+    "m.fs.pervap.cond_duty = Var(m.fs.time, initialize=1, units=pyunits.J/pyunits.s)  # we will calculate this later"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's define our surrogate equations for flux and permeance, and link them to the port variables. Users can use this structure to write custom relations between inlet and outlet streams; for example, here we define the outlet flow of the pervaporation unit as a sum of the inlet flow and calculated recovery fluxes. By defining model constraints in lieu of rigorous mass balances, we add the flux as a custom mass balance term via an empirical correlation and calculate only the condensation duty rather than implementing full energy balance calculations:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Surrogate and first principles model equations\n",
+    "\n",
+    "# flux equation (gas constant is defined as J/mol-K)\n",
+    "\n",
+    "def rule_flux(pervap, t, p, i):\n",
+    "    return pervap.permeate.flow_mol_phase_comp[t, p, i] == (pervap.permeance[t, p, i] *\n",
+    "                                                            exp(-pervap.energy_activation[t, p, i] /\n",
+    "                                                                (Constants.gas_constant *\n",
+    "                                                                 pervap.inlet.temperature[t]))\n",
+    "                                                           ) * pervap.area[t]\n",
+    "m.fs.pervap.eq_flux = Constraint(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list, rule=rule_flux)\n",
```

Comment:
Not important. Technically, this is not the equation for flux, but for the actual permeate flowrate. In order to get the flux, you need to divide the RHS by area. So, if possible, please change the description and/or the name of the equation.